play-media-native/screens/MediaListing.tsx

```diff
@@ -47,15 +42,26 @@ export const MediaListingScreen = ({ navigation }) => {
     refetchMedia();
   }, [refetchMedia]);
 
+  const onCardPress = useCallback(
+    (media: Media) => {
+      navigation.navigate('MediaDetail', { id: media.id, title: media.name });
```

Comment:
Add types please for the MediaDetail screen in interfaces/navigator. Check edit/create screens as an example.

On top of your component, add:
type Props = NativeStackScreenProps<RootStackParamList, 'MediaDetail'>;

and add types for the route params.